### ISSUE TITLE
kubernetes-client introduces deprecations

### ISSUE THREAD

**Issue by lholmquist** [nodeshift/openshift-rest-client#140]
After the latest kubernetes-client update, 279846b1686e6a08cfd3be49652a85f4407d9a8e, some deprecatiions were introduced.  Log of the tests:

```
kubernetes-client deprecated require('kubernetes-client').config, use require('kubernetes-client/backends/request').config. lib/openshift-rest-client.js:25:17
kubernetes-client deprecated require('kubernetes-client').config, use require('kubernetes-client/backends/request').config. index.js:23:39
    ✔ has the OpenshiftClient Object
    ✔ has the config Object

  openshift client tests

kubernetes-client deprecated Client({ config }), see https://github.com/godaddy/kubernetes-client/blob/master/merging-with-kubernetes.md lib/openshift-rest-client.js:89:18
kubernetes-client deprecated Request() without a .kubeconfig option, see https://github.com/godaddy/kubernetes-client/blob/master/merging-with-kubernetes.md node_modules/kubernetes-client/lib/swagger-client.js:199:19
```

At some point we are going to need to update to the new way of doing things

**Comment by lholmquist** [comment]
closing,  fixed in #160

**Comment by lholmquist** [comment]
reopening because of #162

**Comment by lholmquist** [comment]
closed via 9f7b3798813eadd584b685e71f2b657c1cb010be
